styles.css

```diff
@@ -0,0 +1,29 @@
+.main-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+}
+
+.form{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+
+.description{
+    font-weight: bolder;
+}
+
+.label-input{
+    margin-top: 6px;
+}
+
+.continents{
+    margin-top: 6px;
+}
+
+.textarea{
+    margin-top:6px;
+}
```

Comment:
I'm impressed with your proper use of css box containers. but all of your css codes looks disorganised and if you were to make proper use of style linters in this project, it definitely wouldn't pass the linters test. its more adivisable to use linters in your codebase it helps make sure you keep your code clean and organised. 😉 for a more descriptive guide on what linters are and how to use linters in your codebase click  [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md). let me know if you'd need help or more descriptive guide setting it up, you can reach out to me for help if you are facing any diffulty with linters.